.pre-commit-config.yaml

```
@@ -17,3 +17,7 @@ repos:
     rev: v0.9.1
     hooks:
       - id: sphinx-lint
+
+ci:
+  autofix_prs: false
+  autoupdate_schedule: monthly
```

For busier repos, `autofix_prs: false` might be needed, at least before discussion. But here, I think we can try out autofix PRs already.

And `quarterly` is probably just fine to prevent things getting too old.

```suggestion
  autoupdate_schedule: quarterly
```